test/transforms.jl

```diff
@@ -654,6 +654,50 @@
     @test n1 == n2
   end
 
+  @testset "Filter" begin
+    # Test with _check_no_missing_in_row
+    col1 = [1, 2, 3, 4, 5]
+    col2 = [missing, 2, 3, 4, 5]
+    col3 = [5, 5, 5, 5, 5]
+    col4 = [1, 2, 3, 4, 5]
+    col5 = [1, 2, missing, 4, 5]
+    table = Table(;col1, col2, col3, col4, col5)
```

Comment:
Please follow the convention in the other tests. We usually give columns different names x1, x2, ..., please just adopt the convention in the nearby tests.